hack/set-pre-commit.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+mkdir -p ~/.git-templates/hooks
+cp -f ./hack//githooks/commit-msg ~/.git-templates/hooks
```

Comment:
@esequielvirtuoso why the double slash?

Reply:
@ebbauer 

Don't worry, this file has been removed.